lib/plaindate.ts

```diff
@@ -247,13 +248,13 @@ export class PlainDate implements Temporal.PlainDate {
       microsecond = 0,
       nanosecond = 0;
     if (temporalTime !== undefined) {
-      temporalTime = ES.ToTemporalTime(temporalTime);
-      hour = GetSlot(temporalTime, ISO_HOUR);
-      minute = GetSlot(temporalTime, ISO_MINUTE);
-      second = GetSlot(temporalTime, ISO_SECOND);
-      millisecond = GetSlot(temporalTime, ISO_MILLISECOND);
-      microsecond = GetSlot(temporalTime, ISO_MICROSECOND);
-      nanosecond = GetSlot(temporalTime, ISO_NANOSECOND);
+      let temporalTimeConverted = ES.ToTemporalTime(temporalTime);
```

Comment:
Should this be const?

Reply:
If we end up using this pattern I'll change it over to const. I've pushed an alternative that you might like more - PTAL.